### DISCUSSION TITLE
Capital of Vietnam - a debate?

### DISCUSSION THREAD

**Discussion by jeprecated** [axelboc/anki-ultimate-geography#605] [General]
The moment us nerds all wait for happened. Someone asked if I knew the capital of her home country, Vietnam. 

> Ohhh yes 😏 I believe I do, it is Hanoi 🤓 

"No"

> .... what?

"It used to be Hanoi, but it is Ho Chi Minh City"

> W-wait, really? Oh well I guess the deck must be wrong, I should update everyone!

And I check wiki and... it's Hanoi. I look elsewhere and see the same thing.

![image](https://github.com/anki-geo/ultimate-geography/assets/1471350/8ed40705-401c-4e7d-aa41-cb27d2965740)

Am I missing something? 😅 The Vietnamese version of wiki also agrees

**Comment by aplaice** [comment]
Weird. Ho Chi Minh City is what was previously called Saigon, which was the capital of South Vietnam, so _maybe_ that causes the confusion (though the name "Ho Chi Minh City" was given _after_ the fall of South Vietnam), but all sources seem to agree that the current capital of Vietnam is Hanoi and AFAICT this hasn't changed since reunification/conquest of the south.

https://en.wikipedia.org/wiki/List_of_historical_capitals_of_Vietnam confirms this.

(Unless it's a conspiracy that only Vietnamese people are privy to and they're generally not supposed to share it with outsiders?)

**Reply by jeprecated** [reply]
Dun dun dunnnnnn 😱 That must be it 😆
